### PR TITLE
Tiny README.md improvements

### PR DESCRIPTION
**Pull request by EcutDavid**
Hey guys, thanks for the nice work, this PR is just for some tiny README improvement :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,6 @@
 Arizen is ZenCash light wallet.
 
 # User Manuals
-- v1.1.3: [Arizen v1.1.3 Wallet User Manual.pdf](https://github.com/ZencashOfficial/arizen/releases/download/v1.1.3/Arizen.v1.1.3.Wallet.User.Manual.pdf)
```
Comment (EcutDavid) [inline README.md]: This file is not available right now :(

File: README.md [hunk 2]
```diff
@@ -65,7 +64,7 @@ printed. This created address can by included to your Arizen or not.
 ## Development
 
 ### How to create distribution
-npm run dist
+      npm run dist
```
Comment (EcutDavid) [inline README.md]: This change is for aligning with code in other parts of this README.file, like 
image

File: README.md [hunk 3]
```diff
@@ -93,15 +92,15 @@ npm run dist
 - Prefer cloning HTML `<template>`s to constructing DOM trees manually.
 
 #### HTML
-- Do not quote _simple_ HTML attribute values. For example write
+- Do not quote _simple_ HTML attribute values, for example
```
Comment (EcutDavid) [inline README.md]: This README is mostly using the style of `X, for example`

**Comment by EcutDavid** [comment]
One thing I'm not very sure, so didn't included in the code change, what do you guys think?
```
# Arizen
Arizen is ZenCash light wallet.
```
Maybe following is more friendly and encouraging:
```
# Arizen
A ZenCash light wallet implementation.
```

**Comment by EcutDavid** [comment]
Hi, it looks like there is a typo in the repo description
image
"Zencash" -> "ZenCash"?

File: README.md (hunk 1 above)
Comment (LukasBures) [reply README.md]: It will be added today/tomorrow

File: README.md (hunk 2 above)
Comment (LukasBures) [reply README.md]: ack

File: README.md (hunk 3 above)
Comment (LukasBures) [reply README.md]: ack

File: README.md [hunk 4]
```diff
@@ -93,15 +92,15 @@ npm run dist
...
 
       <span id=foo class=bar>
 
   instead of
 
       <span id="foo" class="bar">
 
-  unless you have to, for example in
+  unless you have to, for example
```
Comment (LukasBures) [inline README.md]: ack

**Comment by EcutDavid** [comment]
@LukasBures thanks! Changed :)

**Pull request merged**